Simplified/Utilities/RSAUtils.swift

```diff
@@ -0,0 +1,9 @@
+import CommonCrypto
+
+class RSAUtils {
```

Comment:
didn't we have this also in NYPLAudiobookToolkit? If so can't we just expose and use that?

Reply:
Sure, should I just update the NYPLAudiobookToolkit? Cause it's just adding two `public` mark on the class and function

Reply:
I think making that public in NYPLAudiobookToolkit sounds great. Less code duplication.

Long term, we should think about creating a more generic NYPLUtilities framework... or something that allows us to share general utilities across repos.